package.json

```diff
@@ -331,6 +331,10 @@
             {
                 "fileMatch": ".aws/templates.json",
                 "url": "./out/src/schemas/templates.json"
+            },
+            {
+                "fileMatch": "*task-def.json",
```

Comment:
"task-def.json" seems like too general a file match. I'm concerned about users with non-ECS files getting this schema applied to them. Do you have a more specific file match that you could apply?

Reply:
Would `*ecs-task-def.json` would work for you?

Reply:
Sounds good to me

Reply:
Sweet and done!